Sources/Controllers/2.0/Settings/OAMainSettingsViewController.h

```diff
@@ -16,7 +16,7 @@
 @property (weak, nonatomic) IBOutlet UITableView *settingsTableView;
 @property (weak, nonatomic) IBOutlet UILabel *titleView;
 @property (weak, nonatomic) IBOutlet UIButton *backButton;
-@property (nonatomic) BOOL isShouldBeClosedByBackButton;
+@property (nonatomic) BOOL shouldBeClosedByBackButton;
```

Comment:
What is the point of this variable? You can use OASuperViewController's ```backButtonClicked``` without worrying  about how the controller was presented. (You can connect the button outlet directly to the OASuperViewController's method)